bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -0,0 +1,449 @@
+
+{% block site_header %}
+  {% if LANG.startswith('en-') %}
+    {% include 'includes/global-nav.html' %}
+  {% else %}
+    {% with is_home = True %}
+      {% set logo_src = static('img/pebbles/moz-wordmark-dark-reverse.svg') %}
+      {% include 'mozorg/home/includes/nav.html' %}
+    {% endwith %}
+  {% endif %}
+{% endblock %}
+
+{% macro card_block(class, tag_label, media_icon, title, ga_title, desc, image_url, include_highres_image, video_url, aspect_ratio, link_url, meta) -%}
+<section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if video_url %}has-video-embed{% endif %}">
+  <a class="mzp-c-card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link">
+    <div class="mzp-c-card-media-wrapper">
+      {{ lazy_img(image_url=image_url, include_highres_image=include_highres_image, placeholder_url='placeholder.png', optional_attributes={'class': 'mzp-c-card-image'}) }}
+    </div>
+    <div class="mzp-c-card-content">
+    {% if tag_label %}
+      <div class="mzp-c-card-tag {% if tag_class %}{{ tag_class }}{% endif %}">{{ tag_label }}</div>
+    {% endif %}
+      <h2 class="mzp-c-card-title"><span>{{ title }}</span></h2>
+    {% if desc %}
+      <p class="mzp-c-card-desc">{{ desc|truncatechars(140) }}</p>
+    {% endif %}
+    {% if meta %}
+      <p class="mzp-c-card-meta">{{ meta }}</p>
+    {% endif %}
+    </div>
+  </a>
+  {% if video_url %}
+  <div class="mzp-c-card-video-wrapper hidden">
+    <figure class="mzp-c-card-video-content">
+      <video class="mzp-c-card-video" preload="none" controls playsinline data-ga-label="{{ ga_title }}">
+        <source src="{{ video_url }}.webm" type="video/webm">
+        <source src="{{ video_url }}.mp4" type="video/mp4">
+      </video>
+      <figcaption>
+        <p>{% if desc %}{{ desc }}{% endif %} <a href="{{ link_url }}">{{ _('Read more') }}</a></p>
+      </figcaption>
+    </figure>
+  </div>
+  {% endif %}
+</section>
+{%- endmacro %}
+
+{% block content %}
+<main>
+  <header class="hidden">
+    {# Main page h1 is hidden from view and exists mainly for SEO purposes #}
+    <h1>{{ self.page_title() }}</h1>
+  </header>
+  <section id="download-firefox-primary-cta" class="download-firefox-primary-cta">
+    <div class="mzp-l-content">
+      <div class="primary-wrapper">
+          <h2 class="primary-title">{{ _('The new <strong>Firefox</strong>') }}</h2>
+          <h3 class="primary-title-sub">{{ _('Fast for good.') }}</h3>
+          <p class="primary-desc">{{ _('2x the speed. Built-in privacy protection. Powered by Mozilla.') }}</p>
+          <p class="primary-desc-sub">{{ _('There’s a better way to browse.') }}</p>
+          {{ download_firefox(dom_id='download-primary', download_location='primary cta') }}
+      </div>
+    </div>
+  </section>
+  <section id="download-firefox-sticky-cta" class="download-firefox-sticky-cta">
```

Comment:
This block is mostly a duplicate of the one above, which would be repetitive in a screen reader. Since it's positioned sticky what if we put it at the end of the page in the source order? There's another Firefox promo there but at least the text is different.

Reply:
This isn't new or part of this PR, and I'm not gonna start messing with how this sticky header thing works 😉

Reply:
FWIW it's hidden with JS until it becomes visible.